<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据管理系统</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2"/>
   </item>
   <item row="0" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="passwordBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>修改密码</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="chartBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>销售统计</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="manageBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>数据管理</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="managePage">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="text">
            <string>地理数据管理</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QToolBox" name="toolBox">
             <property name="frameShape">
              <enum>QFrame::WinPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <widget class="QWidget" name="toolBoxPage1">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>338</width>
                <height>363</height>
               </rect>
              </property>
              <attribute name="label">
               <string>出售地理数据</string>
              </attribute>
              <widget class="QLabel" name="label_9">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>200</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>描述：</string>
               </property>
              </widget>
              <widget class="QPushButton" name="sellCancelBtn">
               <property name="geometry">
                <rect>
                 <x>150</x>
                 <y>250</y>
                 <width>75</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>取消</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_6">
               <property name="geometry">
                <rect>
                 <x>21</x>
                 <y>50</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>类型：</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_7">
               <property name="geometry">
                <rect>
                 <x>21</x>
                 <y>87</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>地名：</string>
               </property>
              </widget>
              <widget class="QComboBox" name="geomTypeComboBox">
               <property name="geometry">
                <rect>
                 <x>63</x>
                 <y>50</y>
                 <width>211</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
              <widget class="QComboBox" name="sellBrandComboBox">
               <property name="geometry">
                <rect>
                 <x>63</x>
                 <y>87</y>
                 <width>210</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
              <widget class="QPushButton" name="sellOkBtn">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>250</y>
                 <width>75</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_8">
               <property name="geometry">
                <rect>
                 <x>21</x>
                 <y>160</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>数量：</string>
               </property>
              </widget>
              <widget class="QSpinBox" name="sellNumSpinBox">
               <property name="geometry">
                <rect>
                 <x>63</x>
                 <y>160</y>
                 <width>51</width>
                 <height>22</height>
                </rect>
               </property>
              </widget>
              <widget class="QLineEdit" name="sellSumLineEdit">
               <property name="geometry">
                <rect>
                 <x>63</x>
                 <y>200</y>
                 <width>201</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QLabel" name="label_24">
               <property name="geometry">
                <rect>
                 <x>21</x>
                 <y>120</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>范围：</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="sellPriceLineEdit">
               <property name="geometry">
                <rect>
                 <x>63</x>
                 <y>120</y>
                 <width>211</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QLabel" name="sellLastNumLabel">
               <property name="geometry">
                <rect>
                 <x>130</x>
                 <y>160</y>
                 <width>111</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="text">
                <string>剩余数量：000</string>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="page_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>338</width>
                <height>337</height>
               </rect>
              </property>
              <attribute name="label">
               <string>增加已有地理数据入库</string>
              </attribute>
              <widget class="QLabel" name="label_2">
               <property name="geometry">
                <rect>
                 <x>21</x>
                 <y>50</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>类型：</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_5">
               <property name="geometry">
                <rect>
                 <x>21</x>
                 <y>87</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>地名：</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_10">
               <property name="geometry">
                <rect>
                 <x>21</x>
                 <y>120</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>范围：</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_11">
               <property name="geometry">
                <rect>
                 <x>21</x>
                 <y>160</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>数量：</string>
               </property>
              </widget>
              <widget class="QPushButton" name="dataOkBtn">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>250</y>
                 <width>75</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
              <widget class="QPushButton" name="dataCancelBtn">
               <property name="geometry">
                <rect>
                 <x>150</x>
                 <y>250</y>
                 <width>75</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>取消</string>
               </property>
              </widget>
              <widget class="QComboBox" name="dataTypeComboBox">
               <property name="geometry">
                <rect>
                 <x>63</x>
                 <y>50</y>
                 <width>151</width>
                 <height>20</height>
                </rect>
               </property>
              </widget>
              <widget class="QLineEdit" name="dataPriceLineEdit">
               <property name="geometry">
                <rect>
                 <x>63</x>
                 <y>120</y>
                 <width>211</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QComboBox" name="dataBrandComboBox">
               <property name="geometry">
                <rect>
                 <x>63</x>
                 <y>87</y>
                 <width>210</width>
                 <height>20</height>
                </rect>
               </property>
              </widget>
              <widget class="QLineEdit" name="dataSumLineEdit">
               <property name="geometry">
                <rect>
                 <x>63</x>
                 <y>200</y>
                 <width>211</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="focusPolicy">
                <enum>Qt::ClickFocus</enum>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
              <widget class="QLabel" name="label_20">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>200</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>描述：</string>
               </property>
              </widget>
              <widget class="QSpinBox" name="dataNumSpinBox">
               <property name="geometry">
                <rect>
                 <x>63</x>
                 <y>160</y>
                 <width>61</width>
                 <height>20</height>
                </rect>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="page_5">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>338</width>
                <height>337</height>
               </rect>
              </property>
              <attribute name="label">
               <string>新地理数据入库</string>
              </attribute>
              <widget class="QSpinBox" name="newNumSpinBox">
               <property name="geometry">
                <rect>
                 <x>73</x>
                 <y>140</y>
                 <width>61</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>999</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
              <widget class="QLabel" name="label_14">
               <property name="geometry">
                <rect>
                 <x>31</x>
                 <y>100</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>范围：</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_15">
               <property name="geometry">
                <rect>
                 <x>31</x>
                 <y>140</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>数量：</string>
               </property>
              </widget>
              <widget class="QComboBox" name="newTypeComboBox">
               <property name="geometry">
                <rect>
                 <x>73</x>
                 <y>30</y>
                 <width>151</width>
                 <height>20</height>
                </rect>
               </property>
              </widget>
              <widget class="QLabel" name="label_16">
               <property name="geometry">
                <rect>
                 <x>31</x>
                 <y>30</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>类型：</string>
               </property>
              </widget>
              <widget class="QPushButton" name="newCancelBtn">
               <property name="geometry">
                <rect>
                 <x>160</x>
                 <y>230</y>
                 <width>75</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>取消</string>
               </property>
              </widget>
              <widget class="QLabel" name="label_18">
               <property name="geometry">
                <rect>
                 <x>31</x>
                 <y>67</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>地名：</string>
               </property>
              </widget>
              <widget class="QPushButton" name="newOkBtn">
               <property name="geometry">
                <rect>
                 <x>40</x>
                 <y>230</y>
                 <width>75</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="newBrandLineEdit">
               <property name="geometry">
                <rect>
                 <x>73</x>
                 <y>67</y>
                 <width>210</width>
                 <height>20</height>
                </rect>
               </property>
              </widget>
              <widget class="QLineEdit" name="newSumLineEdit">
               <property name="geometry">
                <rect>
                 <x>72</x>
                 <y>180</y>
                 <width>211</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
              <widget class="QLabel" name="label_22">
               <property name="geometry">
                <rect>
                 <x>30</x>
                 <y>180</y>
                 <width>36</width>
                 <height>16</height>
                </rect>
               </property>
               <property name="text">
                <string>描述：</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="newSumLineEdit_2">
               <property name="geometry">
                <rect>
                 <x>70</x>
                 <y>100</y>
                 <width>211</width>
                 <height>20</height>
                </rect>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="dailyList">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
      <zorder>label</zorder>
      <zorder>toolBox</zorder>
      <zorder>dailyList</zorder>
      <zorder>manageBtn</zorder>
     </widget>
     <widget class="QWidget" name="chartPage">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="text">
          <string>销售统计</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="typeComboBox"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="updateBtn">
         <property name="text">
          <string>更新显示</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>700</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>修改密码</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>120</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>原密码：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>160</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>新密码：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="oldPwdLineEdit">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>120</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="newPwdLineEdit">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>160</y>
         <width>161</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="changePwdBtn">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>220</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
